style.css

```diff
@@ -0,0 +1,7 @@
+h1 {
+    color: red;
```

Comment:
Looks good, heres a suggestion can we utilize the colour blue because...

Reply:
sure! will look into it ASAP